Content.Shared/Roles/JobRequirements.cs

```diff
@@ -126,6 +126,20 @@ public static class JobRequirements
                     }
 
                     var deptDiff = deptRequirement.Time.TotalMinutes - playtime.TotalMinutes;
+                    var nameDepartment = Loc.GetString("role-timer-department-unknown");
+
+                    if (!prototypes.TryIndex<DepartmentPrototype>(deptRequirement.Department, out var departmentName))
+                    {
+                        Logger.Error($"DepartmentPrototype not found in JobRequirement.");
+                    }
+                    else if (!string.IsNullOrEmpty(departmentName.Name))
+                    {
+                        nameDepartment = departmentName.Name;
+                    }
+                    else
+                    {
+                        Logger.Error($"Name of the DepartmentPrototype with the uid {deptRequirement.Department} null or empty in JobRequirement.");
+                    }
```

Comment:
I'm pretty sure this is the wrong logger call anyway and is just adding more obsolete warnings.